.travis.yml

```diff
@@ -76,6 +76,8 @@ matrix:
       - TEST_TARGET="ci-coquelicot"
     - if: NOT (type = pull_request)
       env:
+      # ppx_tools_versioned requires at least this version of findlib
+      - FINDLIB_VER="1.5.0"
```

Comment:
Usually, I would put the variables in a single line. Note that you can also do `FINDLIB_VER=""` as to not to force a concrete version of findlib. This may be more reliable wrt future.